policyengine_us/parameters/gov/states/wi/tax/income/rates/separate.yaml

```diff
@@ -10,7 +10,10 @@ metadata:
       href: https://www.efile.com/wisconsin-tax-brackets-rates-and-forms/
     - title: State of Wisconsin Department of Revenue
       href: https://www.revenue.wi.gov/Pages/FAQS/pcs-taxrates.aspx
-
+    - title: 2023 I-010 Form 1, Wisconsin income tax
+      href: https://www.revenue.wi.gov/TaxForms2023/2023-Form1f.pdf
```

Comment:
Just the instructions should be enough, we only want forms that explicitly mention the rates and brackets